lib/adb.js

```diff
@@ -55,7 +55,7 @@ class ADB {
 
   initJars () {
     const tempJars = [
-      'move_manifest.jar', 'sign.jar', 'appium_apk_tools.jar',
+      'move_manifest.jar', 'sign.jar',
```

Comment:
I think move_manifest.jar could also be safely excluded

Reply:
it also makes sense to rebuild other jars with newer java version to make the notarization process successful

Reply:
Yes, but other jars are no issue for the notarization according to my friend.
So, let me do it as another PR